src/components/search-result-list/search-result-list-item/search-result-list-item.tsx

```diff
@@ -83,32 +86,38 @@ const SearchResultListItem: React.FC<SearchResultListItemProps> = ({
   };
 
   return (
-    // We know that is not following a11y recommendations to have an onclick handler
-    // on a noninteractive element.
+    // We know that is not following a11y recommendations to have an onclick
+    // handler on a non-interactive element.
+    //
     // The reason why this is implemented:
-    // We have interactive elements within each search result: the favourite button,
-    // which must react to clicks
-    // while we also want the entire search result to be clickable.
-    // You cannot have nested links so onClick handlers
-    // and stopping event propagation is necessary.
+    // We have interactive elements within each search result
+    // namely the the favorite button, which must react to clicks while we also want the
+    // entire search result to be clickable.
+    // You cannot have nested links so onClick handlers and stopping event propagation
+    // is necessary.
     //
     // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
     <article
+      ref={listItemRef}
       className="search-result-item arrow arrow__hover--right-small"
       onClick={handleClick}
       onKeyUp={(e) => e.key === "Enter" && handleClick}
     >
       <div className="search-result-item__cover">
-        <SearchResultListItemCover
-          id={manifestationPid}
-          description={String(fullTitle)}
-          url={materialFullUrl}
-          tint={coverTint}
-        />
+        {isVisible && (
+          <SearchResultListItemCover
+            id={manifestationPid}
+            description={String(fullTitle)}
+            url={materialFullUrl}
+            tint={coverTint}
+          />
+        )}
```

Comment:
It would be nice if we could render an "empty" cover e.g. just with the color/tint, without trying to load any data.

This could also work for skeletons. Perhaps one could trigger it if no pid is passed to the element.

Reply:
I don't agree in the visual idea of showing a skeleton item that is partly skeleton partly "real rendering". I like the "full skeleton" expression better. But I think we could ask the customer/designer about this as they are the stakeholders.

I think it works like it is now, where the cover initially shows the tint background and when the cover service returns an image url it is loaded.